syntheseus/interface/reaction.py

```diff
@@ -56,11 +51,18 @@ def unique_reactants(self) -> set[Molecule]:
     def unique_products(self) -> set[Molecule]:
         return set(self.products)
 
+    @property
+    def reactants_combined(self) -> str:
+        return molecule_bag_to_smiles(self.reactants)
+
+    @property
+    def products_combined(self) -> str:
+        return molecule_bag_to_smiles(self.products)
```

Comment:
This was the original name of these properties in `ReactionSample`, but maybe we should call them differently to make it clear that they convert the reactants/products to a string? One could imagine "combining" a `Bag[Molecule]` into a single (disconnected) `Molecule`, which is _not_ what is done here...

Reply:
Yeah I think the names `reactants_str` and `products_str` would be more fitting.